Puppetfile.lock

```diff
@@ -45,13 +45,13 @@ FORGE
       camptocamp-systemd (< 2.0.0, >= 0.4.0)
       puppetlabs-concat (< 5.0.0, >= 1.0.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.13.0)
-    puppet-extlib (2.0.1)
+    puppet-extlib (2.1.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.6.0)
-    puppet-mongodb (2.2.1)
+    puppet-mongodb (2.2.2)
       puppetlabs-apt (< 5.0.0, >= 2.1.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.13.0)
     puppet-staging (3.2.0)
-    puppetlabs-apache (3.1.0)
+    puppetlabs-apache (3.2.0)
```

Comment:
In foreman-installer we've pinned to `3.1.0` so I'd prefer to keep these in sync.

Reply:
I followed the documented instructions to pin new versions. Should this have changed? I didn't do it deliberately.

Reply:
No, it's just a nasty process where the tools aren't exactly designed to do what we want. My goal is to closer align to the tooling but that comes down to merging katello-installer into foreman-installer and a rather big change.